business_requirement_activity_report/report/timesheet_activity_report.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+#   2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.osv import fields, osv
+from openerp import tools
+
+
+class TimesheetReport(osv.Model):

```

Comment:
@victormartinelicocorp I developed it in old API in the first place because I copied/pasted the code from another old API module and I didn't know how to port it to the new API. I think it should be quite easy to port, but this is not the priority.
@elicoidal do you confirm we can keep the old API?